infrastructure/src/hasura/index.ts

```diff
@@ -117,16 +126,27 @@ export class Hasura extends MultiEnvRootStack {
     // ANCHOR Health check
     fargateServicePattern.targetGroup.configureHealthCheck({
       enabled: true,
-      healthyThresholdCount: 3,
-      interval: cdk.Duration.seconds(5),
-      path: '/healthz',
-      timeout: cdk.Duration.seconds(3),
+      healthyThresholdCount: scalingConfig?.healthCheck.healthyThresholdCount ?? 2,
+      interval: cdk.Duration.seconds(scalingConfig?.healthCheck.interval ?? 3),
+      path: scalingConfig?.healthCheck.path ?? '/health',
+      timeout: cdk.Duration.seconds(scalingConfig?.healthCheck.timeout ?? 2),
     })
 
     // ANCHOR Autoscaling
-    const scalableTarget = fargateServicePattern.service.autoScaleTaskCount({ minCapacity: 3, maxCapacity: 50 })
-    scalableTarget.scaleOnCpuUtilization('CpuScaling', { targetUtilizationPercent: 50 })
-    scalableTarget.scaleOnMemoryUtilization('MemoryScaling', { targetUtilizationPercent: 70 })
+    const scalableTarget = fargateServicePattern.service.autoScaleTaskCount({
+      minCapacity: scalingConfig?.autoscaling.autoScaleTaskCount.minCapacity ?? 1, 
+      maxCapacity: scalingConfig?.autoscaling.autoScaleTaskCount.maxCapacity ?? 30 
+    })
+
+    scalableTarget.scaleOnCpuUtilization( 
+      scalingConfig?.autoscaling.scaleOnCpuUtilization.name ?? 'CpuScaling', 
```

Comment:
@igorosip0v it's ok to leave the `CpuScaling` and `MemoryScaling` names intact, no need to move them to params.

Reply:
Updated